renderdoc/driver/vulkan/vk_layer.cpp

```diff
@@ -163,7 +163,7 @@ VkResult VKAPI_CALL hooked_vkCreateInstance(const VkInstanceCreateInfo *pCreateI
                                             const VkAllocationCallbacks *pAllocator,
                                             VkInstance *pInstance)
 {
-  WrappedVulkan *core = new WrappedVulkan("");
+  WrappedVulkan *core = new WrappedVulkan(0);
```

Comment:
Yeh that doesn't quite tie together. That looks like a separate bug, nowhere there does it go through this function that's changed. That stack suggests that somehow the UI hasn't properly transferred the capture over to the remote server before opening and it's trying to open the wrong file.